### PR TITLE
feat: additional testing content

### LINKED ISSUE #76: More Testing Content
@vobu writes more content for testing
@osigge review

### PR DESCRIPTION
**Pull request by vobu**
- context
- automation

### CODE REVIEW

**Comment by vobu** [comment]
still WIP, no need to review yet - will move into proper status once i have everything in

**Comment by marianfoo** [comment]
Hey @vobu 
what is the current status here?
Do you need support in any form?

**Comment by vobu** [comment]
will update with hot-off-the-press `wdi5` info! thx for the reminder

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -233,6 +233,8 @@ GEM
       jekyll-seo-tag (~> 2.1)
     multi_json (1.15.0)
     multipart-post (2.1.1)
+    nokogiri (1.12.5-x86_64-darwin)
```
Comment (marianfoo) [inline docs/Gemfile.lock:236]: @vobu what´s happening here?

Comment (vobu) [reply docs/Gemfile.lock:236]: erm, just your average `bundle install` - "i didn't do anything", but adding some content 🤷

Comment (marianfoo) [reply docs/Gemfile.lock:236]: yeah well, i guess it´s not gonna break things

**Pull request closed without merging**